DataFormats/CaloTowers/interface/CaloTower.h

```diff
@@ -198,6 +207,12 @@ class CaloTower : public reco::LeafCandidate {
    GlobalPoint emPosition_;
    GlobalPoint hadPosition_;
 
+  //hcal limits in CT continuous ieta (from CaloTowerTopology)
+  int firstHBRing_, lastHBRing_;
```

Comment:
thinking of this a bit more: if they are set as transient, then their values will not be recovered when you read the calotowers back. 
Since these values define the behavior of the tower member access methods, the behavior after reading back will change, which is not what's needed here.
The more appropriate way is to make these methods depend on some topology decoder, which will provide a more general solution.

If I understand correctly the values are used to define if the tower is in some specific position "bin" (like "hbRing", "hfRing" etc)
and the number of these bins is limited, possibly defined as an enum or a set of flags in a bitmask
An alternative would be to save  this enum, which is still more effective than 8 ints.